packages/babel-plugin-transform-es2015-classes/src/index.js

```diff
@@ -46,6 +46,10 @@ export default function ({ types: t }) {
         if (state.opts.loose) Constructor = LooseTransformer;
 
         path.replaceWith(new Constructor(path, state.file).run());
+
+        if (path.isCallExpression() && path.get("callee").isArrowFunctionExpression()) {
+          path.get("callee").arrowFunctionToExpression();
```

Comment:
We don't really have a firm policy for when it's cool to insert actual arrows, so I added a few new arrows in place of `.shadow` but explicitly converted them to keep the tests passing.